### PR TITLE
Flipper Admin UI Updates

### PR DESCRIPTION
**Pull request by U-DON**
## Description of change
- Modified Flipper UI to allow us to replace the Flipper views with our own templates.
- Redesigned features list and feature pages with VA.gov design system with clearer messaging on toggle states and usage.

## Testing done
Local testing of Flipper admin site. Interacted with all the different toggles, enabling and disabling.

## Testing planned
Verify changes on deployed environments.

## Acceptance Criteria (Definition of Done)
- [x] Flipper admin site should have VA.gov branding and styling.
- [x] Feature toggles should be explicit about their current states.

#### Applies to all PRs
- [x] Appropriate logging
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)

### CODE REVIEW

**Comment by U-DON** [comment]
Please excuse the inflated change stats... Since the VA.gov design system is not hosted on a CDN, I had to directly pull in the assets.

File: lib/flipper/action_patch.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    def custom_views_path
+      Flipper::UI.configuration.custom_views_path
+    end
+
+    # This is where we store the feature descriptions.
+    # You can choose to store this where it makes sense for you.
+    def yaml_features
+      @yaml_features ||= FLIPPER_FEATURE_CONFIG['features']
+    end
```
Comment (U-DON) [inline lib/flipper/action_patch.rb]: Pulled this in from the previous monkey patch so it's available to the other views in the Flipper UI.

File: config/initializers/flipper.rb [hunk 2]
```diff
@@ -20,12 +21,18 @@
   end
 end
 
-# Labeling what flipper calls "actors" as "users" in the UI
+# Modify Flipper::UI::Configuration to accept a custom view path.
+Flipper::UI::Configuration.prepend(FlipperExtensions::ConfigurationPatch)
+
 Flipper::UI.configure do |config|
+  config.custom_views_path = Rails.root.join('lib', 'flipper', 'views')
```
Comment (U-DON) [inline config/initializers/flipper.rb]: Created a `views` directory within `lib/flipper` to store all our custom Flipper templates.

File: lib/flipper/views/features.erb [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<div class="vads-l-row">
+  <div class="vads-l-col">
+    <table class="vads-u-margin-top--0">
+      <caption class="vads-u-font-size--xl">Features</caption>
+      <thead>
+        <tr>
+          <th>Status</th>
+          <th>Feature</th>
+          <th>Description</th>
+          <th>Enabled Gates</th>
```
Comment (U-DON) [inline lib/flipper/views/features.erb]: Kept this the same for now, but I'm sure there's a better name for this to describe the conditions for enabled features...

File: lib/flipper/views/feature.erb [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+            This feature is <strong>enabled for everyone</strong>.
+          <% elsif @feature.off? %>
+            This feature is <strong>disabled</strong>.
+          <% else %>
+            This feature is <strong>enabled with certain conditions</strong>.
+          <% end %>
+        </p>
+
+        <% if @feature.boolean_value %>
+        <p>Conditional toggles are currently disabled. If you want to conditionally enable this feature, you must first disable the feature for everyone.</p>
```
Comment (U-DON) [inline lib/flipper/views/feature.erb]: This observation came from testing out the toggles; it looks like the main toggle button is the "boolean gate", and it pretty much overrides any conditional toggles. But it doesn't look like there are currently any methods to only disable the boolean gate, so we'll need to disable everything before setting the conditional toggles again in order to activate their effects.

File: lib/flipper/views/features.erb (hunk 3 above)
Comment (rianfowler) [reply lib/flipper/views/features.erb]: This is fine with me.

**Pull request merged**